scripts/run-spark.py

```diff
@@ -159,14 +165,24 @@ def validate_output(args):
             "/terasort/output",
             "/terasort/validate",
         ]
-    ]
+    )
     cmd = " ".join(parts)
     # TODO: pipe logs to teravalidate.log
     return run(cmd)
 
 
-def log_metrics(result):
-    wandb.init(project="spark", entity="raysort")
+def log_metrics(args, result):
+    config = {
+        "total_gb": args.total_gb,
+        "input_part_size": args.input_part_size,
+        "num_workers": args.num_workers,
+        "map_parallelism": args.map_parallelism,
+        "executor_cores": args.executor_cores,
+        "external_shuffle": args.external_shuffle,
+        "push_based_shuffle": args.push_based_shuffle,
+    }
+
+    wandb.init(project="spark", entity="raysort", config=config)
 
     app_id = re.search("application_\d+_\d+", str(result.stderr)).group()
```

Comment:
Better to use r-strings for regex, see https://stackoverflow.com/questions/12871066/what-exactly-is-a-raw-string-regex-and-how-can-you-use-it

Reply:
Fixed.